docs/docs/guides/overview.md

```diff
@@ -2,6 +2,11 @@
 
 The following section describes how to manage Dagger throughout its lifecycle.
 
+### [Choosing a Dagger Source](./guides/choose_source.md)
```

Comment:
link is broken

Reply:
Seems like these links are broken in the .md pages even in [master](https://github.com/odpf/dagger/blob/main/docs/docs/guides/overview.md) branch.

Fixed via commit https://github.com/odpf/dagger/pull/190/commits/49eca4140cc54d6968fec1f6980ea702d78760fd